examples/ota-provider-app/ota-provider-common/OTAProviderExample.cpp

```diff
@@ -279,7 +280,7 @@ EmberAfStatus OTAProviderExample::HandleQueryImage(chip::app::CommandHandler * c
 
     response.status = queryStatus;
     response.delayedActionTime.Emplace(delayedActionTimeSec);
-    if (mUserConsentNeeded && requestorCanConsent)
+    if (mUserConsentNeeded || requestorCanConsent)
```

Comment:
Yes, but the CLI option is described as:
"If provided, and value of RequestorCanConsent field in QueryImage Command is true, then value of UserConsentNeeded field in the QueryImageResponse is set to true.".
So I guess this is to select what to do when the requestor is able to ask for consent. But if it's not able than the provider has no choice but asking for the consent itself rather than relaying that onto the requestor.

Reply:
Got it, fixed it, Please take a look.